### ISSUE TITLE
Error with multiline selectors with Subsitutions

### ISSUE THREAD

**Issue by Tiedye** [microsoft/typescript-styled-plugin#164]
This is valid
```css
  a ${'a'}:first + a {}
```
so is this
```css
  a 
    a:first +
    a {}
```
but this throws an error (colon expected and semicolon expected line 2)
```css
  a 
    ${'a'}:first +
    a {}
```
This is a serious problem because prettier is reformatting selectors in a way that causes this error to manifest

**Comment by MageJohn** [comment]
This is also a problem when shorthand properties with substitutions are broken onto multiple lines. 

This is valid:
```js
styled.div`
  border-top: ${cssVar("line-width-weak")} ${cssVar("line-style-main")} ${cssVar("primary-color")};
`
```

But this is invalid:
```js
styled.div`
  border-top: ${cssVar("line-width-weak")} 
    ${cssVar("line-style-main")} ${cssVar("primary-color")};
`
```

This is despite the resulting CSS being valid. Again, this is a problem because Prettier formats the code in the second way. A workaround in this case could be splitting the shorthand and putting each one on its own line; however for us this violates a stylelint rule that forbids longhand properties when a shorthand could be used.

**Comment by mjbvz** [comment]
Closing as this package has been deprecated in favor of the [official styled-components/typescript-styled-plugin fork]( https://github.com/styled-components/typescript-styled-plugin)

After updating to that fork, follow up in the styled-components repo if this is still an issue
